source/MaterialXTest/CMakeLists.txt

```diff
@@ -121,6 +121,9 @@ set_target_properties(
     VERSION "${MATERIALX_LIBRARY_VERSION}"
     SOVERSION "${MATERIALX_MAJOR_VERSION}")
 
+# Enable Catch benchmarking
+target_compile_definitions(MaterialXTest PRIVATE -DCATCH_CONFIG_ENABLE_BENCHMARKING)
```

Comment:
Shouldn't this step be contingent upon `MATERIALX_BUILD_BENCHMARK_TESTS` being ON, so that we don't increase compile times for standard builds?